cves/CVE-2016-3092.yml

```diff
@@ -208,4 +218,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Input validation was improperly handled originally. Prior to the fix
+    boundary existence was checked, however the boundary being too long was not
```

Comment:
I would like some more expansion on the mistake answer. It feels a bit short and incomplete.